tests/helper.js

```diff
@@ -117,6 +138,26 @@ module.exports = {
       });
   },
 
+  getTextFromElementNative: async (element) => {
+    try {
+      await browser.wait(
+        EC.presenceOf(element),
+        12000,
+        `Element taking too long to appear in the DOM.Let us retry ${element.locator()}`
+      );
+      return await element.getText();
+
+    } catch (error) {
+      await browser.sleep(1000);
+      await browser.wait(
+        EC.visibilityOf(element),
+        12000,
+        `Element taking too long to appear in the DOM. Giving up! ${element.locator()}`
+      );
+      return await element.getText();
```

Comment:
The await here should be able to be removed. The callers to this function are handling the promise by awaiting.

Reply:
Fixed!